### DISCUSSION TITLE
[Script request] Semaphore(Ansible UI)

### DISCUSSION THREAD

**Discussion by Weeb-with-NPC-energy** [community-scripts/ProxmoxVE#68] [Request script]
### Application Name

Semaphore

### Website

https://github.com/semaphoreui/semaphore

### Description

Modern UI and powerful API for Ansible, Terraform, OpenTofu, PowerShell and other DevOps tools.

### Due Diligence

- [X] I have searched existing [scripts](https://community-scripts.github.io/Proxmox/scripts) and found no duplicates.
- [X] I have searched existing [discussions](https://github.com/community-scripts/ProxmoxVE/discussions?discussions_q=) and found no duplicate requests.
- [X] This is not a game-related request.

**Comment by quantumryuu** [comment]
I can take a look at it.
Should we go for BoltDB(included in Semaphore), MySQL or PostgreSQL?

**Reply by Mellowlynx** [reply]
This should probably be published together with the Ansible script when it's done.
(If I remember correctly, someone is working on that already)

**Reply by quantumryuu** [reply]
If no one is doing it, i'm almost done and doing last tests before doing a PR.
Using BoltDB for Semaphore.
@MickLesk

**Reply by quantumryuu** [reply]
@Mellowlynx which Ansible script are you talking about? An LXC with Ansible installed?

**Reply by MickLesk** [reply]
I do nothing with ansible :smile: I can only involve my colleague, he is the nerd in kubernetes and ansible.

**Reply by MickLesk** [reply]
And if we did, I would probably take the default from the script. Otherwise, I'm always Team PostgreSQL, because it's just awesome :smile:

**Reply by quantumryuu** [reply]
Will push a PR with BoltDB(default db of semaphore) :)

**Comment by Mellowlynx** [comment]
Sorry for the late reply, I was sick and did not see the original email, until the closed email I just got :)

But I think this should be an option within an Ansible LXC script.
(Just as the docker one, where you can add portainer as a web-ui add-on.)
Semaphore is also an add-on, and I think it will not work by itself without Ansible on the same host (or within the same LXC in our case)
Not sure about this, might be possible with config changed, did not test it. I installed both myself on an LXC a long time ago, as I do not like complex configs and both in the same LXC is better in that case.

So would be nice if we could add the Ansible script and add Semaphore as an add-on option and after maybe remove the standalone Semaphore script.
